src/org/pentaho/reporting/platform/plugin/JobManager.java

```diff
@@ -64,11 +64,21 @@ public JobManager() {
   }
 
   public JobManager( final boolean isSupportAsync, final long pollingIntervalMilliseconds,
+                     final long dialogThresholdMillisecond, final boolean promptForLocation ) {
+    if ( !isSupportAsync ) {
+      logger.info( ASYNC_DISABLED );
+    }
+    this.config = new Config( isSupportAsync, pollingIntervalMilliseconds, dialogThresholdMillisecond,
+      promptForLocation );
+  }
+
+  public JobManager( final boolean isSupportAsync, final long pollingIntervalMilliseconds,
```

Comment:
Avoid duplicate code (and magically cover more lines in tests) by re-routing this constructor to the one with 4 parameters.